pkg/controller/cassandracluster/reconcile.go

```diff
@@ -659,8 +664,13 @@ func FlipCassandraClusterUpdateSeedListStatus(cc *api.CassandraCluster, status *
 
 					rackName := cc.GetRackName(dc, rack)
 					dcRackName := cc.GetDCRackName(dcName, rackName)
+
 					dcRackStatus := status.CassandraRackStatus[dcRackName]
 
+					if dcRackStatus == nil {
+						return errors.New("DC Rack Status empty")
+					}
```

Comment:
This one is not needed. If the first test errors out that code won't be executed.

Reply:
That's also what I thought until the operator crashed even with 1st nil check :/ (cf. previous message)

Reply:
it doesn't make any sense even with concurrency